packages/server/modules/core/services/streams/clone.ts

```diff
@@ -153,14 +161,16 @@ const cloneStreamObjectsOldFactory =
   }
 
 type CloneStreamObjectsDeps = {
-  db: Knex
+  newProjectDb: Knex
+  sourceProjectDb: Knex
 }
 
 // For sample onboarding stream, goes from 25s to ~250ms vs `cloneStreamObjectsOld`
+// TODO: This kind of query is not supported in multiregion, we can use the old one but apparently its 10 times slower...
```

Comment:
yea, cloneStream is not multiregion ready. it can be made multiregion ready fairly easily, but its gotta use a solution thats apparently (judging by the comment) 10 times slower.

this is only used for onboarding stream cloning which has nothing to do with multiregion, so its fine for now

Reply:
we'll need to implement project move between regions anyhow in a followup task, im happy to park this until then